lib/matplotlib/tests/test_image.py

```diff
@@ -42,6 +42,19 @@ def test_image_interps():
     ax3.imshow(X, interpolation='bicubic')
     ax3.set_ylabel('bicubic')
 
+@image_comparison(baseline_images=['interp_alpha'],
+                  extensions=['png'])
```

Comment:
Looks great.  Please add `remove_text=True` to the image comparison args to reduce our dependence on fonts...